pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -1134,6 +1134,8 @@ type NodeGroupBase struct {
 	VolumeType *string `json:"volumeType,omitempty"`
 	// +optional
 	VolumeName *string `json:"volumeName,omitempty"`
+	// Some AMIs (bottlerocket) have a separate volume for the OS
+	AdditionalEncryptedVolume *string
```

Comment:
this can probably be just a `string` right? (or `[]string` :grin: )

Reply:
🤣  I'll change it to a string